setup_user.py

```diff
@@ -0,0 +1,6 @@
+from assign_rights.models import User
+
+if len(User.objects.all()) == 0:
```

Comment:
this is a fine way to do it, but you could also check to see if that user already exists: 

`if User.objects.filter(username="test").exists():`